helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -532,4 +562,31 @@ service:
         {{- end }}
         {{- end }}
     {{- end }}
+
+    {{- if (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
+    {{- if and (eq .Values.logsEngine "otel") .Values.logsCollection.journaldLogs.enabled }}
+    logs/journald:
```

Comment:
I recommend reusing existing `logs/extraFiles` pipeline instead of creating a new one. Just add journald receiver there if enabled. We can rename that pipeline to `logs/host`

Reply:
Sure, will update it. 
BTW - `logs/extraFiles` pipeline didn't exist at the time of the PR I think :)